analyzer.py

```diff
@@ -0,0 +1,52 @@
+import pickle
+import twitter
+import random
+
+markov = dict()
+
+def generate_markov(messages,num=2):
+    '''
+    Process the input message and generate a markov map for the prefix and surfix.
+    '''
+    global markov
+    for i in range(len(messages)):
+        message = messages[i]
+        current_words = tuple()
+        for word in message.rstrip().split():
+            word = word.strip('@')
+            word = word.strip('#')
```

Comment:
It might be interesting to save #words as more important in some way, considering they usually describe the intent/content of a tweet